src/targets/services/helpers/model.js

```diff
@@ -60,6 +67,14 @@ export class Model {
     return model
   }
 
+  static fromCompressedShares(compressedShares, finalize) {
```

Comment:
It's generally seen as a bad practice to use boolean params in JS and it's event better to make clear it's boolean. Prefer using objects, for instance: `fromCompressedShares(compressedShares, {shouldFinalize})`.
See https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/#2-the-cheap-way-use-a-named-parameter